src/WebJobs.Script.Grpc/MessageExtensions/StatusResultExtensions.cs

```diff
@@ -27,12 +29,18 @@ public static bool IsFailure(this StatusResult statusResult, out Exception excep
             }
         }
 
-        public static bool IsSuccess<T>(this StatusResult status, TaskCompletionSource<T> tcs)
+        public static bool IsInvocationSuccess<T>(this StatusResult status, TaskCompletionSource<T> tcs)
         {
             switch (status.Status)
             {
                 case StatusResult.Types.Status.Failure:
-                    tcs.SetException(GetRpcException(status));
+                    var rpcException = GetRpcException(status);
+                    // TODO - once add protobuf, remove below
```

Comment:
Since all exceptions will pass through the ScriptTelemetryProcessor, I moved the flag to that class. ([ScriptTelemetryProcessor.cs](https://github.com/Azure/azure-functions-host/blob/fe1f3a80dd4d8d5ccc207c398fa88bf891061273/src/WebJobs.Script/Config/ScriptTelemetryProcessor.cs#L23)